static/js/app.js

```diff
@@ -276,10 +276,54 @@ $(function() {
             return params;
         },
         setMultipleMarkersIcon: function() {
+            var markerGroups = [];
+            var markerObj;
+            var groupID = 1;
+            var groupsColours = [];
```

Comment:
Better call it "severities" or something, rather than "colours", since that's what it's called.
(Also, I think we should stick to American spelling for consistency).